src/kaleidoscope/driver/attiny/SAMD.h

```diff
@@ -0,0 +1,225 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::driver::attiny::samd -- ATTiny flashing driver for SAMD MCUs
```

Comment:
"driver::attiny" is wrong. I'm not quire sure what the right name is.

This is is a flashing tool for a custom i2c bootloader we wrote (based on ancient atmel code).

Reply:
`kaleidoscope::util::flash_helper::attiny`?

Reply:
kaleidoscope::util::flasher::keyboardio_i2c_bootloader as mentioned above probably makes sense.